pkgs/development/python-modules/gunicorn/default.nix

```diff
@@ -10,6 +10,10 @@ buildPythonPackage rec {
     sha256 = "fa2662097c66f920f53f70621c6c58ca4a3c4d3434205e608e121b5b3b71f4f3";
   };
 
+  propagatedBuildInputs = [
+    setuptools
+  ];
```

Comment:
Since this is on master now, I would remove this commit, and rebase on master to avoid the massive rebuilds this incurs

Reply:
Hm, looks like it's on `staging` and `staging-next` but not master yet, presume just need to wait a few days

Reply:
you're correct, it hasn't landed in master, thought it was in the last batch of staging-next. I would wait a few days

Reply:
I'll fix locally with overlay for now & rebase